kolyanTask/task.js

```diff
@@ -0,0 +1,68 @@
+const AIRCRAFT_TYPES = {
+    plane: 'plane',
+    copter: 'copter',
+}
+
+class AircraftTypeError extends Error {
+    constructor(message) {
+        super(message);
+        this.name = 'AircraftTypeError';
+    }
+}
+
+class MotorCountError extends Error {
+    constructor(message) {
+        super(message);
+        this.name = 'MotorTypeError'
+    }
+}
+
+class Aircraft {
+    constructor() {}
+    
+    type = null;
+    motors = 0;
+    propeller = {};
+
+    setType(type) {
+        let arrOfTypes = Object.values(AIRCRAFT_TYPES)
```

Comment:
```suggestion
        const arrOfTypes = Object.values(AIRCRAFT_TYPES)
```
if you don't reassign variable use `const`